vocabularies/sample-type.ttl

```diff
@@ -291,12 +311,19 @@ sampty:polished-block a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
 sampty:powdered-chip a skos:Concept ;
-    skos:definition "A rock chip sample that has subsequently been powdered"@en ;
+    skos:definition "A rock chip sample that has subsequently been powdered."@en ;
     skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
     skos:notation "PCHIP" ;
     skos:prefLabel "Powdered Chip"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:pulp a skos:Concept ;
+    skos:definition "A rock sample that has been pulverised sample to a very fine powder and homogenised to be representative of a parent sample, or interval. "@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "PULP" ;
+    skos:prefLabel "Pulp"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
+

```

Comment:
I have not heard the term "pulp" used in geology. Usually, it is crush, grind or mill which is used to describe be the breaking down of rock into a fine grit-free powder.

Reply:
pulp is just a powdered sample and the term is mostly used in exploration and ore deposit geochemistry and assaying

Reply:
If you know of it, then I'm happy :-)